app/views/spree/admin/products/_tax_category_form.html.haml

```diff
@@ -1,5 +1,5 @@
 = f.field_container :tax_category_id do
   = f.label :tax_category_id, t(:tax_category)
   %br
-  = f.collection_select(:tax_category_id, Spree::TaxCategory.all, :id, :name, {:include_blank => Spree::Config.products_require_tax_category ? false : t(:none)}, {:class => "select2 fullwidth"})
+  = f.collection_select(:tax_category_id, Spree::TaxCategory.all, :id, :name, {:include_blank => Spree::Config.products_require_tax_category ? false : t(:none), selected: Spree::TaxCategory.find_by(is_default: true)&.id}, {:class => "select2 fullwidth"})
```

Comment:
This looks right, however, I was wondering if this way users might end up selecting the tax category when it's not required. To prevent this, I think nothing should be selected if the products don't require a tax category. Otherwise, the default category should be selected.

Given that we are incorporating the above logic, is it possible we could move this logic to the view helper to clean up the logic from the view? Thanks.

Reply:
I have the logic requested above (in case Spree::Config.products_require_tax_category = false it shows nothing, otherwise it selects the default category). Don't know where to put the code in the helper tho. Can i put it in tax_helper.rb file in the helpers folder?

Reply:
Moved to a new file (app/helpers/spree/admin/tax_categories_helper.rb) the new logic requested, hope it's within the structure of the project.